idris-erlang.cabal

```diff
@@ -20,13 +20,9 @@ executable idris-codegen-erlang
   other-modules:       IRTS.CodegenErlang
                      , IRTS.CodegenErlang.Foreign
 
-  extensions:          PatternSynonyms
-                     , ViewPatterns
-                     , OverloadedStrings
-
-  build-depends:       base >=4 && <5
+  build-depends:       base             >=4 && <5
                      , containers
-                     , idris >= 0.9.19 && <1
+                     , idris            == 1.*
```

Comment:
Ok, I really hate this, going to change it to `>=1 && <1.1` so we don't break if things change.